src/main/java/com/example/LoyaltyCard.java

```diff
@@ -0,0 +1,29 @@
+package com.example;
+
+import static com.google.common.base.Preconditions;
+
+
+/**
+ * Class implements Value Object.
+ */
+public class LoyaltyCard {
+    /**
+     * Customer's id.
+     */
+    private final String id;
+
+    /**
+     * Constructor for checking id with method checkNotNull.
+     *
+     * @param id is unique number
+     */
+    public LoyaltyCard(String id);
+
```

Comment:
Please fix the code layout to standard Java conventions. It's not going to compile, BTW. You've got `;` at the end.